packages/store-indexer/src/sqlite/apiRoutes.ts

```diff
@@ -44,5 +45,51 @@ export function apiRoutes(database: BaseSQLiteDatabase<"sync", any>): Middleware
     }
   });
 
+  router.post("/q", async (ctx) => {
+    try {
+      const queries = Array.isArray(ctx.request.body) ? ctx.request.body : [];
+      if (queries.length === 0) {
+        ctx.status = 400;
+        ctx.body = JSON.stringify({ error: "No queries provided" });
+        return;
+      }
+
+      const result = [];
+      for (const { query } of queries) {
+        const normalizedQuery = query.trim().toLowerCase();
+        if (!normalizedQuery.startsWith("select")) {
+          ctx.status = 400;
+          ctx.body = JSON.stringify({ error: "Only SELECT queries are allowed" });
+          return;
+        }
```

Comment:
I am guessing I can already break this with `select 1; drop table if exists "some_table"`

Reply:
yeah, I guess there is not much point to it. Looking at dozer as an example, it parses the query as AST, and then selectively checks parts of the tree to make sure it's a valid query but that's quite involved, and still not sure if it couldn't be gamed. I've put it under flag so it's opt-in at least which is perfect for dev environment.